resources/config-defaults.edn

```diff
@@ -95,5 +95,10 @@
  ;; on an application, starting from the first submission.
  :application-deadline-days nil
 
+ ;; List of commands to disable. See the swagger schema of
+ ;; /api/applications/{id} field application/permissions for possible
+ ;; values.
+ :disable-commands []
```

Comment:
Also, based on swagger, I'm not sure if I should write `:disable-commands ["application.command/copy-as-new"]`, `:disable-commands [:application.command/copy-as-new]`, or `:disable-commands [:copy-as-new]` here. Maybe an example would help? (Transforming the string to a keyword is probably already out of reach for most casual users of config.edn without further instructions.)

Reply:
That's a good point. I'll think about it.